src/components/MoneyTemporaryForRefactorRequestConfirmationList.js

```diff
@@ -614,6 +617,217 @@ function MoneyTemporaryForRefactorRequestConfirmationList({
+                    shouldShowRightIcon={!isReadOnly}
+                    title={PolicyUtils.getCleanedTagName(iouTag)}
+                    description={policyTagListName}
+                    numberOfLinesTitle={2}
+                    onPress={() => Navigation.navigate(ROUTES.MONEY_REQUEST_STEP_TAG.getRoute(iouType, transaction.transactionID, reportID, Navigation.getActiveRouteWithoutParams()))}
+                    style={[styles.moneyRequestMenuItem]}
+                    disabled={didConfirm}
+                    interactive={!isReadOnly}
+                    rightLabel={isTagRequired ? translate('common.required') : ''}
+                />
+            ),
+            isRelevant: isTagRequired,
+        },
+        shouldShowTax && {
+            item: (
+                <MenuItemWithTopDescription
+                    key={`${policyTaxRates.name}${taxRateTitle}`}
+                    shouldShowRightIcon={!isReadOnly}
+                    title={taxRateTitle}
+                    description={policyTaxRates.name}
+                    style={[styles.moneyRequestMenuItem]}
+                    titleStyle={styles.flex1}
+                    onPress={() => Navigation.navigate(ROUTES.MONEY_REQUEST_STEP_TAX_RATE.getRoute(iouType, transaction.transactionID, reportID, Navigation.getActiveRouteWithoutParams()))}
+                    disabled={didConfirm}
+                    interactive={!isReadOnly}
+                />
+            ),
+            isRelevant: false,
+        },
+        shouldShowTax && {
+            item: (
+                <MenuItemWithTopDescription
+                    key={`${policyTaxRates.name}${formattedTaxAmount}`}
+                    shouldShowRightIcon={!isReadOnly}
+                    title={formattedTaxAmount}
+                    description={policyTaxRates.name}
+                    style={[styles.moneyRequestMenuItem]}
+                    titleStyle={styles.flex1}
+                    onPress={() => Navigation.navigate(ROUTES.MONEY_REQUEST_STEP_TAX_AMOUNT.getRoute(iouType, transaction.transactionID, reportID, Navigation.getActiveRouteWithoutParams()))}
+                    disabled={didConfirm}
+                    interactive={!isReadOnly}
+                />
+            ),
+            isRelevant: false,
+        },
+        shouldShowBillable && {
+            item: (
+                <View
+                    style={[styles.flexRow, styles.justifyContentBetween, styles.alignItemsCenter, styles.ml5, styles.mr8, styles.optionRow]}
+                    key={translate('common.billable')}
+                >
+                    <Text color={!iouIsBillable ? theme.textSupporting : undefined}>{translate('common.billable')}</Text>
+                    <Switch
+                        accessibilityLabel={translate('common.billable')}
+                        isOn={iouIsBillable}
+                        onToggle={onToggleBillable}
+                    />
+                </View>
```

Comment:
Ouch, it's not a `MenuItem`

Maybe let's switch the naming to "fields"? We have a local variable like `shouldShowAllFields`, so we can be consistent with that, abstracting from specific component names like `MenuItem`.

So `classifiedFields`, `relevantFields`, etc...